### ISSUE TITLE
Handling of multiple backends?

### ISSUE THREAD

**Issue by Nichtraucher** [microg/android_packages_apps_UnifiedNlp#253]
Currently, several different backends are available that work in different ways, some use the same and others use a different data set, and the data is provided in different ways. If more than one backend is installed: Does UnifiedNLP take the measurements from the different backends to make the best out of it?

**Comment by axodentally** [comment]
Yes, UnifiedNLP takes the most accurate measurement, that the services provide. Although a service can provide false information and falsly label it with high accuracy (for example based on wifi APs, that changed location. Happened to me once)

But in recent versions of microG, unifiedNLP is not used anymore.

**Comment by IzzySoft** [comment]
> But in recent versions of microG, unifiedNLP is not used anymore.

Which is a pity. I use different backends for a reason, and would not want to go without my LocalGSM (as it uses no data and thus also works where I have to switch data off as it would otherwise become expensive). I hope there will be a solution for that again. Until then, I rather prefer to stay with versions still supporting UnifiedNlp (and thus those backends).

**Edit:** I just found [this comment by Marvin](https://github.com/microg/GmsCore/issues/1944#issuecomment-1568875751), so my hope to a solution just jumped up a level.
